docker/development/newdle.cfg.docker

```diff
@@ -0,0 +1,91 @@
+# A random string that is secret.
```

Comment:
What do you think about using a minimal config file that only sets stuff that's actually needed (ie no SSO, no exchange config, random availability data, etc) and different from the default and also without all the comments?

I don't want to have another place to update whenever the example config changes.

Reply:
We could also point to the default one, and let users copy it and create a new one. As this one is also committed, I'm not sure how useful it becomes.

Reply:
I guess for a development instance it doesn't really matter... The only alternative to a hardcoded one I see is generating a random one like we do during the non-docker `make`.

I think the whole idea behind being able to run a dev setup in Docker is that you can quickly do some basic development and testing (we still need a command to run the equivalent of `make lint test` in docker!) without having to do much setup... so having to manually create a config kind of defeats this goal.

Reply:
I changed the config file. Wdyt? I think it's the minimum required to be able to run.